jboss/container/eap/galleon/module.yaml

```diff
@@ -5,7 +5,7 @@ description: Install Galleon descriptions and EAP s2i feature-pack. Configure ga
 
 envs:
 - name: S2I_FP_VERSION
-  value: "23.0.0.Final"
+  value: "26.0.0.Final"
```

Comment:
It should be 4.0.0.Final for Xp4, no?

Reply:
+1 fixed, thanks -- not sure why I was confused there.